challenges/prototypes.js

```diff
@@ -36,14 +37,16 @@ function  CuboidMaker(props) {
   this.length = props.length;
   this.width = props.width;
   this.height = props.height
-  this.volume = function () {
-    return (props.length * props.width * props.height);
-  };
-  this.surfaceArea = function () {
-    return 2 * (props.length * props.width + props.length * props.height + props.width * props.height);
-  };
+  
 }
 
+CuboidMaker.prototype.volume = function () {
```

Comment:
You are redeclaring CuboidMaker as well as the volume method.  What is your reasoning for that?